Demo/Configuration.swift

```diff
@@ -187,72 +188,70 @@ internal struct DemoAppSettings: Codable {
 
     internal var cardConfiguration: CardComponent.Configuration {
         var storedCardConfig = StoredCardConfiguration()
-        storedCardConfig.showsSecurityCodeField = cardComponentConfiguration.showsStoredCardSecurityCodeField
+        storedCardConfig.showsSecurityCodeField = cardComponentSettings.showsStoredCardSecurityCodeField
 
         var billingAddressConfig = BillingAddressConfiguration()
-        billingAddressConfig.mode = cardComponentAddressFormType(from: cardComponentConfiguration.addressMode)
+        billingAddressConfig.mode = cardComponentAddressFormType(from: cardComponentSettings.addressMode)
 
-        return .init(showsHolderNameField: cardComponentConfiguration.showsHolderNameField,
-                     showsStorePaymentMethodField: cardComponentConfiguration.showsStorePaymentMethodField,
-                     showsSecurityCodeField: cardComponentConfiguration.showsSecurityCodeField,
-                     koreanAuthenticationMode: cardComponentConfiguration.koreanAuthenticationMode,
-                     socialSecurityNumberMode: cardComponentConfiguration.socialSecurityNumberMode,
+        return .init(showsHolderNameField: cardComponentSettings.showsHolderNameField,
+                     showsStorePaymentMethodField: cardComponentSettings.showsStorePaymentMethodField,
+                     showsSecurityCodeField: cardComponentSettings.showsSecurityCodeField,
+                     koreanAuthenticationMode: cardComponentSettings.koreanAuthenticationMode,
+                     socialSecurityNumberMode: cardComponentSettings.socialSecurityNumberMode,
```

Comment:
We can probably omit word "Component" for `cardSettings` as we do for DropIn and ApplePay?

Reply:
8373462a9f89e37ad62813b71a324ac28cf1670d